extensions/WindowsSandboxExtension/Helpers/Logging.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace WindowsSandboxExtension.Helpers;
+
+public class Logging
+{
+    public static readonly string LogExtension = ".dhlog";
+
+    public static readonly string LogFolderName = "Logs";
+
+    public static readonly string DefaultLogFileName = "wsb";
```

Comment:
Lets update this to `WindowsSandbox `instead.

the sandbox logs would then show up here:

`%userprofile%\AppData\Local\Packages\Microsoft.Windows.DevHome_8wekyb3d8bbwe\TempState\Logs\WindowsSandbox `